guides/user/pipeline/managing-pipelines/index.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: single
+title:  "Managing Pipelines"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Pipelines are the essential tool in Spinnaker for configuring how to deploy your
+application. They are composed of a series of stages that can be combined in
+almost any order, which makes them flexible, consistent, and repeatable.
+
+You can configure your pipelines to run entirely automatically or require manual
+intervention to ensure that everything is working as expected. Similarly,
+pipelines can either be automatically triggered by a wide range of external
+inputs, including other pipelines, or be manually triggered.
+
+This guide explains how to configure and control pipelines, including creating,
+adding triggers to, or disabling pipelines.
+
+## Create a pipeline
+1. Navigate to the **Pipelines** tab in your Spinnaker application.
+  ![](images/pipelines-tab.png)
+2. Click **Create**, located in the upper right corner of the Pipelines tab.
+  ![](images/create.png)
+3. Choose **Pipeline** from the drop down menu and name your pipeline.
+
+After you create your pipeline, add stages to specify the actions that your
+pipeline will perform.
+
+### Add a stage
+
+1. Select **Add stage** from your pipeline configuration screen.
+  ![](images/add-stage.png)
+2. Set the stage type using the drop-down menu.
+3. If this is not the first stage in your pipeline, make sure that this stage
+depends on the desired upstream stage(s) using the **Depends on** field.
+  ![](images/stage-depends-on.png)
```

Comment:
Yes and yes; I removed the note I had about that because @lwander pointed out that "Configuration" is not technically a stage. I don't think that that technicality is worth noting here, though, and I'm open to adding the note about "Configuration" being the first step of every pipeline back in here.

Reply:
Yeah, I think you can mention that it's automatically created and is the first "step" (is a good word for it), and offhandedly mention the technicality that it's not a stage. Up to you tho.

Reply:
Added this back in without directly explaining that Configuration isn't a stage -- let me know what you think.

Reply:
:+1: